### DISCUSSION TITLE
Stats for n locations selected

### DISCUSSION THREAD

**Discussion by F4JSU** [wavelog/wavelog#838] [Ideas]
I mostly do POTA, so I have lots of locations. 1 POTA + 1 date = 1 location.
When I do a rover: 5 POTA on the same day, I create 5 locations.

I'd like to see the statistics for these 5 locations, as in /statistics or Analytics tab.
longest distance, % bands & mode, etc.

It would be interesting to be able to select n locations and get their statistics!

73,
F4JSU

**Comment by HB9HIL** [comment]
Hello @F4JSU 

Did I get you right: 
When you work Park A at a Monday you create a new Location A 
When you work Park B at another day you create a new Location B?

Why? 

You create 1 Station Location for Park A and can use it over and over again. 

If you want to get Stats ONLY for this "activity day" where you activated 5 Parks you Can create 5 locations with the park refs and create a logbook for it called "Special Activity Day with 5 Parks"

Then you link the 5 locations with this Logbook (You can link a location with different logbooks at the same time. To now see the Stats for this logbook simply make it the active one and visit the analytic pages

**Comment by F4JSU** [comment]
Hello @HB9HIL 

> Why?
> You create 1 Station Location for Park A and can use it over and over again.

Sometime I change the setup: power or antenna, and I like to keep a trace.

> Then you link the 5 locations with this Logbook (You can link a location with different logbooks at the same time. To now see the Stats for this logbook simply make it the active one and visit the analytic pages

Less flexible + leave a trace in DB, but can be an alternative, thanks!
